package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@moxy/next-runtime-env",
+  "version": "0.0.0",
+  "description": "NextJS plugin to pass environment variables to NextJS's runtime configuration",
+  "author": "Ivo Lima Silva <ivolimasilva@outlook.com>",
+  "main": "./index.js",
+  "keywords": [
+    "nextjs",
+    "plugins",
+    "environment",
+    "env",
+    "dotenv",
+    "runtime"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:moxystudio/next-runtime-env.git"
+  },
+  "scripts": {
+    "clean": "rm -rf node_modules coverage",
```

Comment:
I don't think we need the clean command.

Moreover, we are missing the commitlint, standard-version, and perhaps other tooling? Check out other MOXY's packages: https://github.com/moxystudio/babel-preset-moxy/blob/master/package.json#L24

Reply:
Removed and added the missing tooling. Will leave this conversation open so you don't forget to check if it's correct now. Thanks.